packages/custom-resource-handlers/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@aws-cdk/custom-resource-handlers",
+  "description": "CDK Custom Resources",
```

Comment:
```suggestion
  "description": "CDK Custom Resources",
  "private": true,
```

Reply:
its at the very bottom but i'll move it up